pages/recipe/create.js

```diff
@@ -46,10 +46,11 @@ export default function CreatePage({ onAddRecipe, recipes }) {
 
   return (
     <>
-      <CreateRecipeForm
+      <CreateOrUpdateRecipeForm
         onHandleSubmit={handleSubmit}
```

Comment:
```suggestion
        onSubmit={handleSubmit}
```
This is about what i mentioned above. Like this the prop `onSubmit` can be used for both `handleSubmit `and `handleUpdateRecipe`